pkg/apis/migration/v1alpha1/directimagestreammigration_types.go

```diff
@@ -76,6 +79,20 @@ func (r *DirectImageStreamMigration) GetDestinationCluster(client k8sclient.Clie
 	return GetCluster(client, r.Spec.DestMigClusterRef)
 }
 
+// Get the DirectImageMigration that owns this DirectImageStreamMigration. If not owned, return nil.
+func (r *DirectImageStreamMigration) GetOwner(client k8sclient.Client) (*DirectImageMigration, error) {
+	owner := &DirectImageMigration{}
+	ownerRefs := r.GetOwnerReferences()
+	if len(ownerRefs) > 0 {
+		ownerRef := types.NamespacedName{Name: ownerRefs[0].Name, Namespace: r.Namespace}
+		err := client.Get(context.TODO(), ownerRef, owner)
+		if err != nil {
+			return nil, liberr.Wrap(err)
+		}
+	}
+	return owner, nil
```

Comment:
Make sure that any code that deals with the returned owner acts sanely if the owner is an empty `DirectImageMigration{}`, since there will be no owner for standalone DISM resources.

Reply:
Fixed